pkg/gds/secrets/token.go

```diff
@@ -8,15 +8,15 @@ import (
 
 var (
 	chars        = []rune("ABCDEFGHIJKLMNPQRSTUVWXYZabcdefghjkmnpqrstuvwxyz1234567890")
-	specialChars = []rune("#$%&()*+-<=>?@[]^_{|}~")
+	specialChars = []rune("#%&()*+-<=>?@[]^_{}")
 )
 
 // CreateToken creates a variable length random token that can be used for passwords or API keys.
 func CreateToken(length int) string {
```

Comment:
Was thinking about using this or similar to generate the CSRF token, but it is not crypto random secure; it's going to be removed soon anyway in favor of admin logins.